src/pages/index.js

```diff
@@ -0,0 +1,72 @@
+import Head from 'next/head';
+import { Header } from '../components/Header';
+
+import styles from '../styles/home.module.scss';
+
+export default function Index() {
+  return (
+    <>
+      <Head>
+        <title>Brasil API</title>
+      </Head>
+      <Header />
+
+      <main className={styles.mainContainer}>
+        <section className={styles.sectionHeader}>
+          <div className={styles.content}>
+            <h2>Transformando o Brasil em uma API</h2>
+            <p>
+              Este projeto experimental tem como objetivo centralizar e
+              disponibilizar endpoints modernos com baixíssima latência
+              independente de sua fonte.
+            </p>
+            <a href="https://brasilapi.com.br/docs">
+              <button type="button">Começar agora</button>
+            </a>
+          </div>
+          <img
+            src="/images/api-schema.svg"
+            className={styles.schemaDesk}
+            alt="API Schema"
+          />
+        </section>
+        <section className={styles.requests}>
+          <div>
+            <h2>Requests</h2>
```

Comment:
Ainda não sei se seria um texto ideal ou se tem algum termo melhor para 
```suggestion
            <h2>Estatísticas</h2>
```

Reply:
é, pode ser sim